config/checkstyle-input-checks.xml

```diff
@@ -24,12 +24,12 @@
 
   <module name="TreeWalker">
     <module name="Regexp">
-      <property name="id" value="StyleValidationCommentInInputs"/>
```

Comment:
https://github.com/search?q=repo%3Acheckstyle%2Fcheckstyle%20StyleValidationCommentInInputs&type=code

Old suppressions need to be removed.

Reply:
Done:
```bash
➜  checkstyle git:(master) ag -Q "StyleValidationCommentInInputs" | wc -l
20


```

```bash
➜  checkstyle git:(issue-13213) ag -Q "StyleValidationCommentInInputs" | wc -l
0


```